.github/workflows/e2e-workflow.yml

```diff
@@ -134,7 +134,7 @@ jobs:
         run: |
           make docker-build-adapter
         env:
-          REGISTRY: ${{ env.REGISTRY }}
+          REGISTRY: ${{ env.CLUSTER_NAME }}.azurecr.io
```

Comment:
Currently this is working unchanged for e2e tests outside of publish-gh-image.yaml